# LuxOS: archiv zapichnuteho commitu qt-solutions mel kontrolni soucet SKIP -> sha512 archivu
# (GitHub archiv commitu a8dda66d, stazeny dvakrat - github.com i codeload.github.com - shodny)
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -14,7 +14,7 @@
         'qtsingleapplication-qupzilla.patch'
         'qtsingleapplication.prf'
         'qtsinglecoreapplication.prf')
-sha512sums=('SKIP'
+sha512sums=('1de4cc3c066a0192397f03db76476ea70e80e17d9a5118fcff1c7111d893876f8d0a6d77d27338342c2f469d697c1876942a226174bb104be7691e1fcf26032d'
             'a459cc42e98cd6b1e58d464fb28861ed4453b9d2f96a11cd8ac68e06c74083ef05452aabe08c4625229aa19de865522cec3cb994504e9eff796ee1033bc38ad2'
             'd8b2b27380a407b84a461fa93ca68751f503a495200e2204b576e6f27b9768ab6e77a1a47c63c90fa5803a9a055b7c0546f8ae4aa9f0fa5cc5969af3e889d4f2'
             '9495604598221ed94a0d5be486468bc0fc936bbe8a22d3c7c23738d48e67a5516bd496e24b7678c1a05ed7b8e3cd0578d6272629e23cd74c4a360f20cf3b663b'
